db/migrate/20230503132014_create_posts.rb

```diff
@@ -0,0 +1,15 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.references :user, null: false, foreign_key: true
```

Comment:
- - Great work on your migration files, however, I would recommend you use `author_id` when referencing the `user` in your posts table as it is in the ERD diagram  👍🏽